ptx/sec_deriv_intro.ptx

```diff
@@ -186,7 +186,17 @@
         <caption>The function <m>f(t)</m> and its secant line corresponding to <m>t=2</m> and <m>  t=3</m></caption>
           <image xml:id="img_derivfalling">
           <!-- START figures/fig_derivfalling.tex -->
-            <description></description>
+          <shortdescription>
+            A decreasing concaved down graph of function  <m>f(t)</m> with its secant line corresponding to <m>t=2 </m> and <m>t=3</m>.
```

Comment:
the phrase 'convave down' is not introduced until Section 3.4, so here you might say something like "a downward-curving graph, along with a secant line between two points on the graph"
Note that the short description should not have math tags in it.